src/KafkaFlow.Abstractions/IEvent.cs

```diff
@@ -9,9 +9,9 @@
     public interface IEvent
     {
         /// <summary>
-        /// Subscribes to the event.
+        /// Subscribes to the async event.
```

Comment:
Maybe we could leave the summary as before.

Reply:
Done, please review